connectors/es/sink.py

```diff
@@ -365,6 +374,14 @@ async def run(self, generator, job_type):
         except asyncio.CancelledError:
             self._logger.info("Task is canceled, stop Extractor...")
             raise
+        except asyncio.QueueFull as e:
+            self._logger.critical("Throttled by Elasticsearch", exc_info=True)
```

Comment:
```suggestion
            self._logger.error("Throttled by Elasticsearch", exc_info=True)
```

critical should be for something that is crashing the whole process. This is just failing the job.

Reply:
Good catch, it's left there from development/debugging :)